crates/abq_cli/src/args.rs

```diff
@@ -318,6 +318,16 @@ pub enum Command {
         #[clap(long, default_value = "7")]
         batch_size: NonZeroU64,
 
+        /// How ABQ will distribute the tests.
+        ///
+        /// By default, ABQ will distribute the next unassigned test to a worker asking for tests.
+        /// work-strategy=by-file, abq will distribute whole-files-at-a-time to workers asking for tests.
+        /// The non-default behavior can be helpful if you have very expensive setup or teardowns in a test file
+        /// that would, by default, be run multiples times by multiples workers instead of a single time on the worker
+        /// responsible for that test file.
+        #[clap(long, default_value = "by-test")]
```

Comment:
I guess, eventually, might be nice to set these at runtime so we can use the `default` values